src/FlightBooking.Core/Entities/Flight.cs

```diff
@@ -0,0 +1,21 @@
+﻿using FlightBookingApp.Core.Common;
+
+namespace FlightBookingApp.Core.Entities
+{
+    public class Flight : BaseAuditableEntity
+    {
+        public DateTime StartDateTime { get; set; }
+
+        public DateTime EndDateTime { get; set; }
+
+        public string? DeparturePoint { get; set; }
```

Comment:
If Departure/Destination points are strings, you can encounter a problem, when user enters incorrect (inexisting, bad-spelling, duplicate) point. I suggest you to create distinct entity for location and use it as a reference for flight departure/destination points. Also I suggest you to look at some sites implementing same functionality to analyze their bad and good solutions